src/maker/api.rs

```diff
@@ -91,7 +91,7 @@ impl Maker {
         wallet_file: &PathBuf,
         rpc_config: &RPCConfig,
         port: u16,
```

Comment:
Make this `Option<u16>`. Use the default port value when its None. That's a more cleaner API.